_pages/en_US/godmode9-usage.txt

```diff
@@ -122,18 +122,52 @@ Insert the game cartridge you intend to dump into your device
   + **NDS Game Cartridge:** Press (A) on `[TitleID].trim.nds` to select it, then select "Copy to 0:/gm9/out"
 1. Your installable `.cia` or non-installable `.nds` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
+## Dumping a Game Cartridges romfs/exefs
+
+1. Insert a game cartrige you wish to dump from
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Hover over "GAMECART ()"
+1. Press (A), wait (could take up to 20 seconds)
+1. Hover over the bigest file ending with `.3ds` and press (A)
+1. Select "NCCH image options"
+1. Select "Mount image to drive"
+1. Press (A) to continue
+1. Hover over `exefs.bin`, hold (R) and press (A) to open "directory options", select "copy to 0:/gm9/out", do the same with `romfs.bin`
+  + The dumped files will be in `/gm9/out`
+
 ## Dumping a Title
 
 1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
 1. Hover over the drive applicable to the type of title you wish to dump:
   + **User Installed Title**: `[A:] SYSNAND SD`
   + **System Title**: `[1:] SYSNAND CTRNAND`
 1. Hold (R) and press (A) at the same time to open the drive options
-1. Select "Search for titles"
+1. Select "Open title manager"
 1. Press (A) to continue
 1. Press (A) on the `.tmd` file to select it, then select "TMD file options...", then select "Build CIA (standard)"
 1. Your installable `.cia` formatted file will be outputted to the `/gm9/out/` folder on your SD card
 
+## Dumping a Titles romfs/exefs
```

Comment:
Any similar comments made for "dumping a cartridge's romfs/exefs" are also relevant here